addons/resource/views/resource_views.xml

```diff
@@ -109,6 +109,7 @@
                         <h1>
                             <field name="name"/>
                         </h1>
+                        <field name="company_id" groups="base.group_multi_company"/>
```

Comment:
Actually I will merge res copany and res company into a single res dropshipping model.